frontend/src/scenes/data-management/events/eventDefinitionsTableLogic.ts

```diff
@@ -209,6 +245,16 @@ export const eventDefinitionsTableLogic = kea<
                 }
             }
         },
+        '/events/stats/:id': ({ id }) => {
+            if (props.syncWithUrl) {
+                if (!values.eventDefinitions.results.length && !values.eventDefinitionsLoading) {
+                    actions.loadEventDefinitions()
+                }
+                if (id) {
+                    actions.setOpenedDefinition(id)
+                }
+            }
+        },
```

Comment:
This scheme seems a bit wonky because it _can't_ be accurate: you can have more than one definition open at the same time. Also, this scheme isn't used at any time when opening definitions manually, and when closing a definition opened based on the URL, the URL stays the same. I propose using a search param instead, as it's more OK to be lax with those.

Reply:
I would totally agree in any other case, but in this case I'm leaning towards keeping the urls as is to remain backwards compatible.